sympy/simplify/trigsimp.py

```diff
@@ -1164,13 +1164,15 @@ def _futrig(e, **kwargs):
             factor_terms, TR12(x), trigs)],  # expand tan of sum
         )]
     e = greedy(tree, objective=Lops)(e)
-
     return coeff*e
 
 
 def _is_Expr(e):
     """_eapply helper to tell whether ``e`` and all its args
     are Exprs."""
+    from sympy import Derivative
+    if isinstance(e, Derivative):
+        return _is_Expr(e.expr)
```

Comment:
@asmeurer apparently `_is_Expr` is creating troubles simplifying the expressions. It blocks the application of the rules if any sub-arg is a non-Expr Basic. It sounds like a dumb condition.

Reply:
It's probably there to avoid issues with things like Piecewise and meijerg that have non-Expr args. But I think in general things should be written in such a way to ignore such expressions (just pass through).

Reply:
In fact, given how many `TypeError: Tuple has no ...` errors I see all the time from using PIecewise, I expect this change to Derivative will root out a lot of these issues all over the place. Hopefully they can be fixed correctly, and not just by special-casing Derivative.

Reply:
Worth testing SymPy without it.